osgtest/tests/test_150_xrootd.py

```diff
@@ -58,7 +63,7 @@ def test_01_start_xrootd(self):
                             owner="xrootd",
                             chown=(user.pw_uid, user.pw_gid))
 
-            files.append(core.config['xrootd.config'], XROOTD_CFG_TEXT % sec_protocol, owner='xrootd', backup=True)
+            files.append(core.config['xrootd.config'], XROOTD_CFG_TEXT % (sec_protocol, core.config['xrootd.port']), owner='xrootd', backup=True)
```

Comment:
Can we split this up so it fits within the 120 char limit?

Reply:
Done